community/modules/file-system/nfs-server/README.md

```diff
@@ -34,6 +37,37 @@ volume as an external file system.
 > you will need to manually delete the disk after destroying a deployment group
 > with `nfs-server`.
 
+## Mounting
+
+To mount the NFS Server you must first ensure that the NFS client has been
+installed the and then call the proper `mount` command.
+
+When mounting to a Slurm resource, a `vm_instance`, or a `batch-job-template`,
```

Comment:
I would point to the compatibility matrix instead of listing out modules here. Now we also have HTCondor and PBS Pro., and it will just be easier to keep a single source of truth up to date.

Reply:
Same thing for filestore and pre-existing-network-storage.

Reply:
Done